tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -22,4 +22,9 @@
     <suppress files=".+org.openhab.binding.yeelight.+" checks="OutsideOfLibExternalLibrariesCheck" />
```

Comment:
Is this file supposed to be present ?

Reply:
This is to suppress false positives from the code-checker.

Reply:
Ok. - fine for me